# Destroy's a problem instance pointed to by problemid
int32 id
---
